src/docs/product/integrations/issue-tracking/incidentio/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Incident.io
+sidebar_order: 1
+redirect_from:
+  - /product/integrations/incidentio/
```

Comment:
I learned the hard way we need a redirect because we link to this from the integration install page. Without adding this it 404s - the only variable is the integration name. E.g. https://sentry.io/settings/hb-meowcraft/sentry-apps/height/ click "Documentation" goes to https://docs.sentry.io/product/integrations/height/

Reply:
Okay so I was confused until I looked in the app. The link in the docs is pointing to the redirect URL rather than just pointing to the actual URL of the newly created page. Is that Documentation link URL being generated vs directly set when the incident.io integration is being added in the app? I would have assumed that when we add a new integration, we link to the URL of the newly added page in docs. The page it's linking to has never existed, so it's weird to me that we're setting it up that way.

Reply:
good to know. thanks for clarifying